app/templates/custom-elements/remote-screen.html

```diff
@@ -30,9 +30,15 @@
     :host([fullscreen="true"]) #remote-screen-img.full-height {
       height: 100%;
     }
+
+    #mobile-keyboard-input {
+      position: fixed;
+      bottom: -1000px;
+    }
   </style>
   <div class="screen-wrapper">
     <img id="remote-screen-img" src="/stream?advance_headers=1" />
+    <input type="text" id="mobile-keyboard-input" />
```

Comment:
On Android, it tries to autocapitalize on every keystroke because it's at the beginning of the text field. We can fix this by adding `autocapitalize="off"` to the `<input>` element.